### PR TITLE
Implementing map modifiers

### PR DESCRIPTION
**Pull request by MatejNota**
Hello, I am working on this issue, and I want to know, if I am on the right track. 

Main change I did was in generateFight.ts -> there I added backroundEffect() in which I highlighted 2 main modifiers, which are tied to certain map:

In swamp (map 10) I halved speed of brutes.
In nether (map 2) I lowered hp of brutes (I want to tweak stats in the future so brutes start at 80% hp instead of 100%)

Then I did ` : weightedRandom(mapModifiers);` , which in the current state has no effect. But it is placeholder for weightd random function, which will be choosing from lesser effects. I am thinking something along the line like +1 speed & -1 strength and similar effects. However I don't know if I should also apply these effects in boss fights and tournaments. My opinion is they should not be applied.

Regarding frontend, I didn't look for sprites yet, but I was thinking in after I do all modifiers, they should be displayed in battle in a small square next to stats. They will have short name with description of changes applied. 

I would like to know your opinion.

### CODE REVIEW

File: prisma/package.json
Comment (Zenoo) [inline prisma/package.json:1]: This file shouldn't be modified, as there is no DB changes yet

File: server/.env.sample
Comment (Zenoo) [inline server/.env.sample:1]: This file shouldn't be deleted

**Review by Zenoo — CHANGES_REQUESTED** [review]
The brute stat changes should be handled in getFighters, like any other skill induced stat change.

Also, I don't understand your mapModifiers, as you are linking odds to maps?
The map selection is already done and weighted, so all that is needed is to lower the odds for the maps that will have modifiers, to make them rare.

Also, anytime this map appears, modifiers should be applied, for whatever fight

**Comment by MatejNota** [comment]
Regarding mapmodifier I was thinking this:

If there is map, where I want fixed effect, handle that immediatelly. If there is map which does not have effect tied to it, do weighted random from several effects. 

But anyway, thanks for feedback. I will start working on fixing this.

**Pull request closed without merging**